### PR TITLE
MAT-4230 Endpoint to fetch multiple valusets 

### PR DESCRIPTION
**Pull request by adongare**

### CODE REVIEW

File: src/main/java/cms/gov/madie/terminology/controller/VsacController.java [hunk 1]
```diff
@@ -38,28 +39,35 @@ public ResponseEntity<String> getValueSet(
...
 
     String serialized = serializeFhirValueset(fhirValueSet);
 
     return ResponseEntity.ok().body(serialized);
   }
 
-  protected String serializeFhirValueset(ValueSet fhirValueSet) {
-
-    String serialized = parser.encodeResourceToString(fhirValueSet);
+  @PutMapping(path = "/value-sets/searches", produces = MediaType.APPLICATION_JSON_VALUE)
```
Comment (sb-cecilialiu) [inline src/main/java/cms/gov/madie/terminology/controller/VsacController.java:54]: @adongare It's nice work Ashok! I'm not sure if this is supposed to replace the current work of getting value set one by one from front end, if it is, I think the ideal solution would be handling the failed value sets and get them mapped with elm standard errors so front end can get both success value sets as well as ready to use failed ones.

Comment (adongare) [reply src/main/java/cms/gov/madie/terminology/controller/VsacController.java:54]: Thanks @sb-cecilialiu! We can modify this lil bit to replace current work but I'm not fully aware of that requirements. I can work with you to understand it a little bit more and draft s story or work on it.

**Review by RohitKandimalla — APPROVED** [review]

File: src/main/java/cms/gov/madie/terminology/controller/VsacController.java (hunk 1 above)
Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/terminology/controller/VsacController.java:54]: @adongare Sure thanks so much!!

**Review by sb-cecilialiu — APPROVED** [review]
Looks good Ashok, thanks!

**Pull request merged**